Model/Order/CompleteOrder.php

```diff
@@ -68,13 +68,16 @@ public function execute(OrderInterface $order): void
         $websiteId = (int)$order->getStore()->getWebsiteId();
         $publicOrderId = $this->getOrderPublicId($order);
         $orderId = $order->getEntityId();
+        $url = sprintf(self::COMPLETE_ORDER_URL, $publicOrderId);
+
         $body = [
-            'platform_order_id' => $orderId,
+            'state' => 'order_complete',
+            'platform_order_id' => $order->getIncrementId(),
+            'platform_friendly_id' => $order->getEntityId()
         ];
-        $url = sprintf(self::COMPLETE_URL, $publicOrderId);
-        $response = $this->client->post($websiteId, $url, $body);
-        if ($response->getStatus() !== 200) {
-            $this->logger->error(__('Failed to complete order with id="%1"', $orderId));
+        $response = $this->client->put($websiteId, $url, $body);
+        if ($response->getStatus() !== 201) {
+            throw new LocalizedException(__('Failed to post order completion with id="%1"', $$orderId));
```

Comment:
We should not throw an exception. The inability to update the order status on the Bold side should not block the order creation.

Reply:
^So in theory this should never fail, if it does I think we should block because the platformIds won't exist in our system and I'm not sure how releasing funds would work. Also, if we didn't have this exception in the first place this would have failed silently and we wouldn't have realized that the order wasn't being updated as expected. 🤔 
Let me know thoughts on this though.